qemu/tests/virtio_fs_share_data.py

```diff
@@ -741,6 +741,14 @@ def check_security_label(file, folder, xattr_name):
                     time.sleep(1)
                     check_security_label(file_share_in_guest, fs_dest, selinux_xattr_name)
 
+                    error_context.context("The list of xattr for the file is empty "
+                                          "in guest, let's check it.", test.log.info)
+                    getfattr_list_cmd = params.get("getfattr_list_cmd")
+                    s, o = session.cmd_status_output(getfattr_list_cmd % file_new_in_guest)
+                    if s:
+                        test.fail("Getting the emply list of xattr failed"
+                                  " on virtiofs fs, the output is %s" % o)
+
```

Comment:
Hi @xiagao 
I didn't get the point of  ```For some file in virtiofs who has no xattr, when getting the xattr, virtiofs should not output any error info.```
From the codes changed, I guess those are to add a step to get the attribute `xattr` of the empty file, right?

Reply:
The point is for the file created on host, while it has no extend attribute on guest, but when we try to check it with "getfattr" cmd inside guest, before it will prompt error info, the expected result is PASS for 'getfattr' cmd.

From the codes changed, I guess those are to add a step to get the attribute xattr of the empty file
--> No, it's not the empty file, just the file has no xattr(extend attribute), it can be shown because of the mapping of security.selinux set on host.